TKGame/Debug.cs

```diff
@@ -150,12 +235,55 @@ private static void UpdateKeyboardOverlay()
             // For each key that is in the dictionary, check to see if it's
             // pressed. If it is, make it the active color, if it isn't, make it
             // the inactive color.
+            KeyboardState keyboardState = Keyboard.GetState();
+
             foreach (var keyLabelPair in keyboardLabelDict)
             {
-                keyLabelPair.Value.TextColor = (Keyboard.GetState().IsKeyDown(keyLabelPair.Key))
-                    ? DEBUG_KEYBOARD_OVERLAY_ACTIVE_COLOR
-                    : DEBUG_KEYBOARD_OVERLAY_INACTIVE_COLOR;
+                Label keyLabel = keyLabelPair.Value;
+
+                keyLabel.TextColor = (keyboardState.IsKeyDown(keyLabelPair.Key))
+                    ? KEYBOARD_OVERLAY_ACTIVE_TEXT_COLOR
+                    : KEYBOARD_OVERLAY_INACTIVE_TEXT_COLOR;
+
+                keyLabel.Background = (keyboardState.IsKeyDown(keyLabelPair.Key))
+                    ? ActiveBackgroundBrush
+                    : InactiveBackgroundBrush;
             }
         }
+
+        /// <summary>
+        /// Draw bounding rectangle around a given rectangle.
+        /// </summary>
+        public static void DrawBoundingRectangle(SpriteBatch spriteBatch, Rectangle rectangle, Color color, int lineWidth)
+        {
+            if (HitboxTexture is null)
```

Comment:
We will need to attach hitboxes to entities later on  probably, so that they can draw their own hitbox only if they exist and based on the info already in the Entity class.